specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -306,35 +401,36 @@ http://contoso-app.azurewebsites.net/login
 @doc("The input artifacts for the test.")
 model TestInputArtifacts {
```

Comment:
Q: do we support TestInputArtifacts only on the Test resource or also on the TestRun resource. Do we need TestRunInputArtifacts model?

Reply:
Treating Test and TestRun Input artifacts as separate entities